backend/controller/console.go

```diff
@@ -44,15 +44,14 @@ func visitNode(sch *schema.Schema, n schema.Node, verbString *string) error {
 	return schema.Visit(n, func(n schema.Node, next func() error) error {
 		switch n := n.(type) {
 		case *schema.Ref:
-			decl, ok := sch.Resolve(n).Get()
-			if !ok {
-				return fmt.Errorf("failed to resolve %s", n)
-			}
-			*verbString += decl.String() + "\n\n"
-			err := visitNode(sch, decl, verbString)
-			if err != nil {
-				return err
+			if decl, ok := sch.Resolve(n).Get(); ok {
+				*verbString += decl.String() + "\n\n"
+				err := visitNode(sch, decl, verbString)
+				if err != nil {
+					return err
+				}
```

Comment:
@alecthomas this change sort of undoes the change that was made here: https://github.com/TBD54566975/ftl/commit/3c91886259a0f6fb88ebd81db6f85ecd9105e5ee#diff-cf10aad402fc38d803d69ee88de91fa8b7f27d33ea52e2dc67ebb6749dfe61afR47-R50 so I wanted to call it out. 

Maybe there's a better approach we can use here, but wanted to fix the broken console until then.

Reply:
This looks identical to me, just reordered, or am I missing something?

Reply:
yeah it's very similar. The difference is that this one doesn't error if the `decl` cannot be resolved, it just skips it. This was happening for `Body` in the `HttpRequest<Body>` data types. Since `Body` isn't a ref that it can find.

Reply:
So instead of:
```go
return fmt.Errorf("failed to resolve %s", n)
```
it just moves on to the next `Ref`

Reply:
Aaah I see, makes sense